src/c++/perf_analyzer/genai-perf/genai_perf/export_data/json_exporter.py

```diff
@@ -26,29 +26,33 @@
 
 
 import json
-from argparse import Namespace
 from enum import Enum
-from pathlib import Path
 from typing import Dict
 
-from genai_perf.constants import DEFAULT_OUTPUT_DATA_JSON
+import genai_perf.logging as logging
+
+DEFAULT_OUTPUT_DATA_JSON = "profile_export_genai_perf.json"
+
+logger = logging.getLogger(__name__)
 
 
 class JsonExporter:
     """
     A class to export the statistics and arg values in a json format.
     """
 
-    def __init__(self, stats: Dict, args: Namespace, extra_inputs: Dict):
-        self._stats = stats
-        self._args = dict(vars(args))
-        self._extra_inputs = extra_inputs
+    def __init__(self, config: Dict):
+        self._stats: Dict = config["stats"]
+        self._args = dict(vars(config["args"]))
+        self._extra_inputs = config["extra_inputs"]
+        self._output_dir = config["artifact_dir"]
         self._stats_and_args: Dict = {}
         self._prepare_args_for_export()
         self._merge_stats_and_args()
 
-    def export_to_file(self, output_dir: Path) -> None:
-        filename = output_dir / DEFAULT_OUTPUT_DATA_JSON
+    def export(self) -> None:
```

Comment:
This 4-line function is more along the lines for what I would expect the other Class's export functions to look like after refactoring.